packages/web/src/uiUtils.ts

```diff
@@ -224,3 +225,22 @@ export function scrollIntoViewIfNeeded(element: Element | undefined) {
 
 const kControlCodesRe = '\\u0000-\\u0020\\u007f-\\u009f';
 export const kWebLinkRe = new RegExp('(?:[a-zA-Z][a-zA-Z0-9+.-]{2,}:\\/\\/|www\\.)[^\\s' + kControlCodesRe + '"]{2,}[^\\s' + kControlCodesRe + '"\')}\\],:;.!?]', 'ug');
+
+export function useFlash(): [boolean, EffectCallback] {
```

Comment:
IIUC, this is only needed to re-trigger the animation mid-flight? Otherwise, we'd just set the css class when needed, and the animation will play by itself?

If so, please add a comment explaining this. I am not 100% sure it's worth the complexity, but I leave that up to you.

Reply:
The important bit is that the css class is removed after a second, so the animation can be played multiple times.

I've added a comment to explain that.